pkg/apis/build/v1alpha1/build_pod.go

```diff
@@ -365,14 +365,27 @@ func (b *Build) setupSecretVolumesAndArgs(secrets []corev1.Secret, filter func(s
 			MountPath: fmt.Sprintf(SecretPathName, secret.Name),
 		})
 
-		annotatedUrl := secret.Annotations[DOCKERSecretAnnotationPrefix]
-		secretType := "docker"
-		if secret.Annotations[GITSecretAnnotationPrefix] != "" {
+		var (
```

Comment:
Can we inline these? These are not used outside of the scope of their respective if blocks.

Reply:
good catch